src/lib/util/domains/domains.ts

```diff
@@ -0,0 +1,38 @@
+//- Library Imports
+import { ethers } from 'ethers';
+
+export const rootDomainId = ethers.constants.HashZero;
+
+const getSubnodeHash = (parentHash: string, labelHash: string): string => {
+	const calculatedHash = ethers.utils.keccak256(
+		ethers.utils.defaultAbiCoder.encode(
+			['bytes32', 'bytes32'],
+			[ethers.utils.arrayify(parentHash), ethers.utils.arrayify(labelHash)],
+		),
+	);
+	return calculatedHash;
```

Comment:
Can be simplified down to `return ethers.utils.keccak256(...)`

Reply:
👍